tests/test_image.py

```diff
@@ -3,5 +3,5 @@
 def test_png_lilypond():
     a = converter.parse('tests/sample.musicxml').write('lilypond.png')
 def test_png_mscore():   
-    #b = converter.parse('tests/sample.musicxml').write('musicxml.png')
-    pass
\ No newline at end of file
+    #b = converter.parse('tests/ActorPreludeSample.musicxml').write('musicxml.png')
```

Comment:
Why is this line commented and this test case not tested???

Reply:
i was just investigating why `converter.parse('*.musicxml').write` never passes on travis pytest. Changed back now